### PR TITLE
Various doc fixes and one cmake default fix

### PR DESCRIPTION
**Pull request by tronical**

### CODE REVIEW

File: docs/building.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ to run.
 To use Slint from C++, the following extra dependencies are needed:
 
 - **[cmake](https://cmake.org/download/)** (3.19 or newer)
-- A C++ compiler that can do C++20 (e.g., **MSVC 2019 16.6** on Windows)
+- A C++ compiler that can supports C++20 (e.g., **MSVC 2019 16.6** on Windows)
```
Comment (ogoffart) [inline docs/building.md:39]: Maybe we should mention a minimum supported version of GCC and clang? I think it's GCC 10

**Review by ogoffart — APPROVED** [review]

File: docs/building.md (hunk 1 above)
Comment (tronical) [reply docs/building.md:39]: Sounds good. I'm unsure about clang, but I'll add gcc 10.

File: docs/building.md [hunk 2]
```diff
@@ -28,25 +28,22 @@ For Linux a few additional packages beyond the usual build essentials are needed
...
-Node.JS support is not built by default! Check below for the extra commands
-to run.
-
 ### C++ API (optional)
 
 To use Slint from C++, the following extra dependencies are needed:
 
 - **[cmake](https://cmake.org/download/)** (3.19 or newer)
-- A C++ compiler that can do C++20 (e.g., **MSVC 2019 16.6** on Windows)
+- A C++ compiler that can supports C++20 (e.g., **MSVC 2019 16.6** on Windows)
```
Comment (hunger) [inline docs/building.md:39]: "that supports", not "that can supports"

You said MSVC2022 above, do you need to adapt this here as well?

Comment (tronical) [reply docs/building.md:39]: Hm, I didn't change it because that's only a requirement when using Skia. But I might as well bump it...

**Pull request merged**